spec/support/devise_macros.rb

```diff
@@ -2,7 +2,7 @@
 module DeviseMacros
   def login_user
     before(:each) do
-      qa_id = '0030P00001y3eLHQAY'
+      qa_id = '0030P000020bdxYQAQ'
```

Comment:
this seems to be a change made necessary by the refresh, but not sure because these files hadn't been updated at all in years.